src/index.js

```diff
@@ -22,12 +22,7 @@ const setBannerExtraParameter = (adUnitId, key, value) => {
 }
 
 const setBannerLocalExtraParameter = (adUnitId, key, value) => {
-  if ((value !== null) && (value !== undefined) && (typeof value !== 'string')) {
-    console.warn("setBannerLocalExtraParameter() supports only string values: " + value);
-    return;
-  }
-
-  AppLovinMAX.setBannerLocalExtraParameter(adUnitId, key, value);
+  AppLovinMAX.setBannerLocalExtraParameter(adUnitId, {key:value});
```

Comment:
Do we really need to wrap it in a map? Why can't we just pass `value` through to the native layer? Can the native layer declare a generic parameter like `Object` (Java) or `id` (iOS) instead of `ReadableMap`

Reply:
It has to be wrapped with a map.   Actually,  I used `Object` / `id` first. Then I got an error of "Got unknown argument class: Object" from [here](https://github.com/facebook/react-native/blob/54033fd3349aff6f4878ce85f814f9a34d7af8e3/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/JavaMethodWrapper.java#L306), which is documented in [Android](https://github.com/facebook/react-native/blob/54033fd3349aff6f4878ce85f814f9a34d7af8e3/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/JavaMethodWrapper.java#L306) and [iOS](https://reactnative.dev/docs/native-modules-ios#argument-types). Arbitaral data can be sent only as a JavaScript object (JSON) to the native side.